src/Resources/config/services.yaml

```diff
@@ -8,3 +8,6 @@ services:
 
     Becklyn\Hosting\Git\GitIntegration:
         $projectDir: '%kernel.project_dir%'
+
+    Becklyn\Hosting\Project\ProjectVersion:
+        $cache: '@cache.app.simple'
```

Comment:
Unfortunately we can't use this service, as it has been added alongside the default implementation for the `PsrCacheInterface`: https://github.com/symfony/symfony/commit/1b6ec8b0ae841cc036c46fd5865a8c060af39881

I still haven't figured out which cache service to use in Symfony 3.4. It's kind of a mess.

Reply:
Okay, I just implement PSR-6